### PR TITLE
scaffolding to accommodate database changes

### PR DESCRIPTION
**Pull request by ekm6**

### CODE REVIEW

File: Benchmark.WebApi/Benchmark.WebApi/Benchmark.DbMigrations/Benchmark.DbMigrations.csproj [hunk 1]
```diff
@@ -1,23 +1,20 @@
...
   </ItemGroup>
-
   <ItemGroup>
     <Folder Include="Functions\" />
     <Folder Include="StoredProcedures\" />
     <Folder Include="Views\" />
   </ItemGroup>
-
-</Project>
+  <ItemGroup />
```
Comment (tonysulfaro) [inline Benchmark.WebApi/Benchmark.WebApi/Benchmark.DbMigrations/Benchmark.DbMigrations.csproj:19]: was this file supposed to change at all?

File: Benchmark.WebApi/Benchmark.WebApi/Benchmark.WebApi/Controllers/EmployeesController.cs [hunk 2]
```diff
@@ -10,7 +10,7 @@
 namespace Benchmark.WebApi.Controllers
 {
     [Route("api/[controller]")]
-    [Authorize]
+   [Authorize]
```
Comment (tonysulfaro) [inline Benchmark.WebApi/Benchmark.WebApi/Benchmark.WebApi/Controllers/EmployeesController.cs:13]: can we move this over one space plz

File: Benchmark.WebApi/Benchmark.WebApi/Benchmark.WebApi/Controllers/ProjectsController.cs [hunk 3]
```diff
@@ -1,22 +1,27 @@
...
 using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
 using System.Collections.Generic;
 using System.Threading.Tasks;
+using System.Web.Http;
 
 namespace Benchmark.WebApi.Controllers
 {
-    [Route("api/[controller]")]
+    [Microsoft.AspNetCore.Mvc.Route("api/[controller]")]
```
Comment (joeldevries) [inline Benchmark.WebApi/Benchmark.WebApi/Benchmark.WebApi/Controllers/ProjectsController.cs:13]: Why are these all fully qualified now?

File: Benchmark.WebApi/Benchmark.WebApi/Benchmark.WebApi/Controllers/ProjectsController.cs [hunk 4]
```diff
@@ -1,22 +1,27 @@
...
+    [Authorize]
     [ApiController]
     public class ProjectsController : BaseUtilityController
     {
         private readonly IProjectManagementService _projectManagementService;
+        private readonly DbContextOptions<BenchmarkContext> _options;
 
-        public ProjectsController(IProjectManagementService projectManagementService)
+        public ProjectsController(IProjectManagementService projectManagementService, DbContextOptions<BenchmarkContext> options)
```
Comment (joeldevries) [inline Benchmark.WebApi/Benchmark.WebApi/Benchmark.WebApi/Controllers/ProjectsController.cs:21]: Why do we need DbContextOptions in a controller? Those would almost always be required only in the repository.

**Pull request merged**